### ISSUE TITLE
Feat: Design FitMate Website Landing Page

### ISSUE THREAD

**Issue by SHITIZ-AGGARWAL** [Code-Sauce-Official/FitMate#41]
**What is your issue related to ?**

- [ ] Code
- [x] Designing
- [ ] Documentation
- [ ] Testing/Research
- [ ] Others[please specify]

**Description**

We are planning to make a website for FitMate. This will consist of FitMate trailer, features , about FitMate, and a footer.
You have to design the website on Figma/Adobe XD/Sketch according to your preference. 

- Make Userflow
- Make Wireframes
- Make final design

**Additional context**
You can add more features but discuss in the comments first

**Comment by subhasmitapradhan** [comment]
I would like to work on this issue.

**Comment by yatharthmago01** [comment]
Yeah sure go ahead!

**Comment by subhasmitapradhan** [comment]
Will I be designing the whole website or just the landing page?

**Comment by yatharthmago01** [comment]
Just design the landing page according to the issue details for now. If and when you have further ideas regarding the website, ping us.

**Comment by sowmiyeh** [comment]
I would like to work on this, please assign me this

**Comment by SHITIZ-AGGARWAL** [comment]
@subhasmitapradhan Update

**Comment by SHITIZ-AGGARWAL** [comment]
@sowmiya-venkatesan Assigning you

**Comment by SHITIZ-AGGARWAL** [comment]
@sowmiya-venkatesan @subhasmitapradhan We have to start the website development soon so if you both can work together or submit your design faster, that will be great.

**Comment by subhasmitapradhan** [comment]
Sure, will do that @SHITIZ-AGGARWAL . @sowmiya-venkatesan can you ping me on the discord. I could not find you or can you lemme know your Id?

**Comment by SHITIZ-AGGARWAL** [comment]
@sowmiya-venkatesan @subhasmitapradhan Any update on this

**Comment by sowmiyeh** [comment]
@SHITIZ-AGGARWAL Yes i've completed with my user flow and Subhasmitapradhan working on her userflow

You can find my design on here -> https://www.figma.com/file/5AuVxTWo7PwJ9no43rwYtO/Fitmate-Website?node-id=0%3A1
